gunicorn/workers/workertmp.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, cfg):
         try:
             if not IS_CYGWIN:
                 util.unlink(name)
-            self._tmp = os.fdopen(fd, 'w+b', 1)
+            self._tmp = os.fdopen(fd, 'w+b', 0)
```

Comment:
It would be nice to add a comment to explain why we pass 0 here as explained in https://github.com/benoitc/gunicorn/issues/2091#issuecomment-522344617.

Reply:
thank you for you suggestion, i try to add a comment for explain